File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        HashSet<int> hMap = new HashSet<int>();
        foreach(int i in nums) {
            if(hMap.Contains(i)){
                return true;
            }
            hMap.Add(i);
        }
        return false;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int currentCount = 0;
        for(int i = 0; i < nums.length; i++){
            if(nums[i]==1){
                currentCount++;
            } else {
                if(max < currentCount) max = currentCount;
                currentCount = 0;
            }
        }
        if(max < currentCount) max = currentCount;
        return max;
    }
}